### PR TITLE
Implemented conversation popups in the homepage newsfeed.

### PR DESCRIPTION
**Pull request by whsieh**

### CODE REVIEW

File: app/assets/stylesheets/homePage/home.less [hunk 1]
```diff
@@ -2,10 +2,17 @@
 @import "../mixins.less";
 @import "homePageStyle.less";
 
+.vertical-scroll-disabled {
```
Comment (impguard) [inline app/assets/stylesheets/homePage/home.less]: This should not be in global LESS scope. Should most likely be in homePageStyle, mixins, or locally within the container.

File: app/assets/html/home.html [hunk 2]
```diff
@@ -1,19 +1,29 @@
-<div id="home-container">
+<div id="home-container" class="{{homeScrollableClass()}}">
     <div id="newsfeed-container">
-        <div id="newsfeed-heading" class="header">Trending Resolutions</div>
+        <div id="newsfeed-heading" class="header" ng-style="{opacity: newsfeedHeaderOpacity()}">Trending Resolutions</div>
```
Comment (impguard) [inline app/assets/html/home.html]: Maybe make these styles classes and swap them with ng-class.

**Comment by impguard** [comment]
+1 after changes made. I like how fast you are a hammering these out now lel.

File: app/models/conversation.rb [hunk 3]
```diff
@@ -25,6 +25,28 @@ def self.recent_with_resolutions(max_num_conversations)
...
+        user1_id = arena.user1_id
+        user2_id = arena.user2_id
+        content = posts.map { |post|
+            {
+                :name => post.author.id == user1_id ? user1_name : user2_name,
+                :text => post.text
+            }
+        }
+        content.reverse!
+        if resolution_state == "in_progress" || resolution_state == :in_progress
```
Comment (leohtkam) [inline app/models/conversation.rb]: if resolution_state.to_s == "in_progress"

**Comment by leohtkam** [comment]
+1 after it passes travis

File: app/models/conversation.rb (hunk 3 above)
Comment (whsieh) [inline app/models/conversation.rb]: Fixed.

**Pull request merged**